### PR TITLE
Update contained list accessibility.mdx

### PR DESCRIPTION
**Pull request by mbgower**
First draft of new accessibility content (without images)

### CODE REVIEW

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Let me know if you need help with images for this PR.

**Review by tay1orjones — APPROVED** [review]
I'm not sure what's up with the preview but the images look good locally.

image

**Review by guidari — APPROVED** [review]

File: src/pages/components/contained-list/accessibility.mdx [hunk 1]
```diff
@@ -9,18 +9,101 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
...
+other accessibility considerations, some of which are described below.
+
+### Keyboard interactions
+
+The default contained list is not interactive, but several of its variants
+include keyboard operation. In all interactive variants, the `Tab` key is used
+for navigation and both `Space` and `Enter` are used to activate components.
+
+Users tab between any actionable items in the list, regardless of whether each
+item is itself clickable or contains an action button (such as ‘delete’). It is
```
Comment (laurenmrice) [inline src/pages/components/contained-list/accessibility.mdx:35]:
```suggestion
item is clickable or contains an action button (such as ‘delete’). It is
```

**Review by laurenmrice — CHANGES_REQUESTED** [review]
I have revised the images in the[ image box folder](https://ibm.ent.box.com/folder/223115679697)

**Comment by mbgower** [comment]
@laurenmrice images and editorial changes incorporated. Please review and approve

**Review by laurenmrice — APPROVED** [review]

**Pull request merged**